packages/common/keys/src/dxn.ts

```diff
@@ -44,6 +44,7 @@ export type QueueSubspaceTag = (typeof QueueSubspaceTags)[keyof typeof QueueSubs
  * ```
  */
 export class DXN {
+  // TODO(burdon): Rename to DXN (i.e., DXN.DXN).
```

Comment:
I don't think that's a good idea

Reply:
let's discuss. main thing is we need the non-FQ variant.